gazebo/rendering/RenderEngine.cc

```diff
@@ -22,6 +22,8 @@
 
 // Not Apple or Windows
 #if not defined(__APPLE__) && not defined(_WIN32)
+#define GL_GLEXT_LEGACY
+#define GLX_GLXEXT_LEGACY
```

Comment:
This code is only related to Linux, why it is needed in this PR that from what I understand is Windows only?

Reply:
I will try to remove it.

Reply:
done.